README.md

```diff
@@ -9,37 +15,43 @@ For more details about the design and architecture of the SDK, see the [official
 ### Required Prerequisites
 To use this SDK you must have:
 
-* **A Java 8 development environment**
+* **A Java development environment**
 
-  If you do not have one, go to [Java SE Downloads](https://www.oracle.com/technetwork/java/javase/downloads/index.html) on the Oracle website, then download and install the Java SE Development Kit (JDK). Java 8 or higher is recommended.
+You will need Java 8 or later. On the Oracle website, go to [Java SE
+Downloads](https://www.oracle.com/technetwork/java/javase/downloads/index.html), and then download and install the Java SE Development Kit (JDK).
 
-  **Note:** If you use the Oracle JDK, you must also download and install the [Java Cryptography Extension (JCE) Unlimited Strength Jurisdiction Policy Files](http://www.oracle.com/technetwork/java/javase/downloads/jce8-download-2133166.html).
+If you use the Oracle JDK, you must also download and install the [Java Cryptography Extension (JCE) Unlimited Strength Jurisdiction Policy Files](http://www.oracle.com/technetwork/java/javase/downloads/jce8-download-2133166.html).
 
 * **Bouncy Castle**
 
-  Bouncy Castle provides a cryptography API for Java. If you do not have Bouncy Castle, go to https://bouncycastle.org/latest_releases.html, then download the provider file that corresponds to your JDK. Or, you can pick it up from Maven:
+Bouncy Castle provides a cryptography API for Java. If you do not have Bouncy Castle, go to [Bouncy
+Castle Latest Java Releases](https://bouncycastle.org/latest_releases.html) to download the provider
+file that corresponds to your JDK. If you use [Apache Maven](https://maven.apache.org), Bouncy
```

Comment:
We should be pulling this in as a transitive dependency - i.e., it'll be automatically pulled in if you use maven to obtain the encryption SDK.